test/theta_sketch_dup_test.cc

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <gtest/gtest.h>
+#include <iomanip>
+#include <fstream>
+#include "theta_sketch_dup.hpp"
+
+namespace datasketches {
+
+TEST(THETA_SKETCH_DUP, TEST) {
+    auto a=update_theta_sketch_dup::builder().set_lg_k(12).build();
+    for (int i=0; i<1000000; i++) a.update(i);
+    for (int i=0; i<10; i++) {
+      a.remove(0);
```

Comment:
keep removing 0 or i?

Reply:
I will update this test file. Actually, the tests need to be modified in general as you suggested in the first comment.